### PR TITLE
Story/q20-1

### PR DESCRIPTION
**Pull request by aa-kashk**
Enable full simple-user capabilities

- user registration
- email password reset

### CODE REVIEW

**Comment by jdamerow** [comment]
Seriously Aakash, please read our guidelines regarding GitHub naming!

**Comment by jdamerow** [comment]
Please resolve conflicts. I've added something to the pom and also fixed the indentation. Please set the right indentation as described here: https://diging.atlassian.net/wiki/spaces/DIGING/pages/2057011201/Java+Code+Style

File: quadriga/pom.xml [hunk 1]
```diff
@@ -19,10 +19,17 @@
...
         <db.database.url>jdbc-url-to-db</db.database.url>
         <db.user>db-user</db.user>
         <db.password>db-password</db.password>
+		
+		<email.user></email.user>
+ 		<email.password></email.password>
+ 		<email.host></email.host>
+ 		<email.port></email.port>
+ 		<email.from></email.from>
+ 		<app.url></app.url>
```
Comment (jdamerow) [inline quadriga/pom.xml]: please adjust indentation

File: quadriga/src/main/java/edu/asu/diging/quadriga/config/SimpleUsersConfig.java [hunk 2]
```diff
@@ -1,14 +1,37 @@
...
+import org.springframework.context.annotation.PropertySource;
 
 import edu.asu.diging.simpleusers.core.config.SimpleUsers;
 import edu.asu.diging.simpleusers.core.config.SimpleUsersConfiguration;
 
 @Configuration
+@PropertySource({"classpath:config.properties", "${appConfigFile:classpath:}/app.properties"})
 public class SimpleUsersConfig implements SimpleUsersConfiguration {
+	
+	@Value("${email_user}")
```
Comment (jdamerow) [inline quadriga/src/main/java/edu/asu/diging/quadriga/config/SimpleUsersConfig.java]: indentaiton

File: quadriga/src/main/webapp/WEB-INF/views/login.html [hunk 3]
```diff
@@ -0,0 +1,42 @@
+<html layout:decorate="~{layouts/main}">
+   <head>
+      <title>CORD19 Geo Explorer</title>
```
Comment (jdamerow) [inline quadriga/src/main/webapp/WEB-INF/views/login.html]: title

File: quadriga/src/main/webapp/WEB-INF/views/password/change.html [hunk 4]
```diff
@@ -0,0 +1,23 @@
+<!DOCTYPE html>
+ <html layout:decorate="~{layouts/main}">
+ <head>
+ <title>Authority Matcher</title>
```
Comment (jdamerow) [inline quadriga/src/main/webapp/WEB-INF/views/password/change.html]: title

File: quadriga/src/main/webapp/WEB-INF/views/resetPassword.html [hunk 5]
```diff
@@ -0,0 +1,33 @@
+<html layout:decorate="~{layouts/main}">
+<head>
+<title>Authority Matcher</title>
```
Comment (jdamerow) [inline quadriga/src/main/webapp/WEB-INF/views/resetPassword.html]: title

File: quadriga/src/main/webapp/WEB-INF/views/resetRequestSent.html [hunk 6]
```diff
@@ -0,0 +1,14 @@
+<html layout:decorate="~{layouts/main}">
+ <head>
+ <title>Authority Matcher</title>
```
Comment (jdamerow) [inline quadriga/src/main/webapp/WEB-INF/views/resetRequestSent.html]: title

**Review by jdamerow — CHANGES_REQUESTED** [review]

**Comment by jdamerow** [comment]
Make it so, Jenkins.

**Comment by jdamerow** [comment]
Make it so, Jenkins.

**Comment by jdamerow** [comment]
There is something weird going on with the password field. It only shows one character:
image

**Comment by jdamerow** [comment]
Make it so, Jenkins.

**Comment by jdamerow** [comment]
The label fields should not look like input fields:
image
And the error message if the passwords don't match should be in a red alert box or something.

Also, let's remove the register icon in the menu. The "sign up" link on the login page should be enough for now.

**Comment by aa-kashk** [comment]
I ran this on my local and checked in both safari and chrome but the input fields are not in search boxes. Can you recheck?
image

**Comment by jdamerow** [comment]
It's on the password reset page.

**Pull request merged**